src/StaticPages/TeamPage.module.scss

```
@@ -0,0 +1,112 @@
+@import "src/components/style/foundation/spacing";
+@import "src/components/style/foundation/colors";
+
+.team {
+  // border: 1px solid color(gray, lighter);
+  width: 1000px;
```

If you enforce the width here, it'll break mobile.
```suggestion
  max-width: 1000px;
```